src/theme/themes/plh_facilitator_mapa/_index.scss

```diff
@@ -80,6 +103,13 @@
       paragraph-margin-medium: 12px,
       paragraph-margin-large: 16px,
 
+      // BLUE PALETTE
+      color-accent-blue-40: var(--ion-color-primary-700),
+      color-accent-blue-50: var(--ion-color-primary-600),
+      color-accent-blue-60: var(--ion-color-primary-500),
+      color-accent-blue-70: var(--ion-color-primary-400),
+      color-accent-blue-80: var(--ion-color-primary-300),
+      color-accent-blue-90: var(--ion-color-primary-200),
```

Comment:
nit(blocking): These should be removed, as I believe you have (sensibly) removed reference to these intermediate variables and used the `--ion-color-primary-*` vars in the styles instead.